### DISCUSSION TITLE
Remove popup message when hovering an annotation

### DISCUSSION THREAD

**Discussion by yjcui** [sumatrapdfreader/sumatrapdf#3864] [Ideas]
First great thanks for this awesome pdf reader!

With the release of 3.5, whenenver I hover over an annotation, a message would pop up on top left corner. It won't affect any function, but kinda distracting/annoying. Is it possible to disable it through configuration? Or remove it from Sumatra itself?

![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/87352042/2791f616-f40a-411b-9fd2-693b6c904142)

**Comment by kjk** [comment]
Maybe will add delay of 100-300 ms so that it doesn't show up immediately.
The reason for the pop-up is to improve discoverability of annotations editing i.e. tell people it's possible.

**Reply by yjcui** [reply]
Thanks for the response! Here are my further thoughts. 
1. Would it be better to completely get rid of this message? First, more often users 'add' annotations than 'edit' them. Second, annotation editing is possible with right click already. If someone really needs to edit, they might have already Googled for this very basic function. It might not be a good idea to distract majority of users with this 'possibility' (besides, too many other 'possibilities' in this SW, ^_^).
2. If you still plan to keep it, maybe with a delay, is it possible to leave it configurable with ON/OFF?
3. Speed and simplicity are the key features for my using Sumatra. To be honest, this pop-up is the only reason I'm still sticking to 3.4.6 now.

Just my two cents, and again thanks for this great SW!
